Example/Swift Example/View Controllers/OAuthViewController.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  OAuthViewController.swift
+//  SKYKit - Swift Example
+//
+//  Copyright 2015 Oursky Ltd.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+import SKYKit
+
+class OAuthViewController: UITableViewController {
+
+    @IBOutlet weak var providerLabel: UILabel!
+    @IBOutlet weak var userEmailLabel: UILabel!
+    @IBOutlet weak var userIDLabel: UILabel!
+    @IBOutlet weak var userAccessTokenLabel: UILabel!
+    @IBOutlet weak var userLastLoginLabel: UILabel!
+
+    let actionSectionIndex = 1
+    let loginProviderIndex = 0
+    let linkProviderIndex = 1
+    let selectedProvider = "google"
+    let dateFormatter = DateFormatter()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.dateFormatter.locale = Locale.current
+        self.dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZZZZZ"
+        self.dateFormatter.timeZone = TimeZone(secondsFromGMT: 0)
+        self.providerLabel.text = selectedProvider
```

Comment:
Is it possible to use `SKYDataSerialization.m`?

Reply:
SKYDataSerialization is supposed to be internal. Should we open it for external use?